Convex_hull_3/examples/Convex_hull_3/quickhull_3_SM.cpp

```diff
@@ -0,0 +1,32 @@
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
```

Comment:
The name of the example is confusing: you call it "quickhull_3_SM.cpp" but the change with "quickhull_3.cpp" is that you passed the polyhedron_3 directly to `convex_hull_3()`...